practice-two/src/styles/layouts/banner.scss

```diff
@@ -60,10 +56,13 @@
 }
 
 @media (min-width: 1440px) {
+  @function toRem($pxValue) {
```

Comment:
Should create only one function for this. Put it on the utils or something like that and reuse it everywhere you need